core/modules/field/field.tokens.inc

```diff
@@ -123,7 +123,7 @@ function _field_token_info($field_name = NULL) {
 
   $cache = cache('token');
 
-  if (!isset($fields)) {
+  if (!isset($info)) {
```

Comment:
Wow, that's a nice catch! :+1:

Reply:
Agreed, nice! Seems that might have a small performance boost in some situations.